CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing
+
+## Content publishing checklist
+When you are ready for your pull request to be reviewed, make sure to go through this list so the reviewer can focus on giving feedback on the content and not have to nitpick.
+
+- [ ] Captalised B for protocol
+- [ ] Lowercase b for currency
```

Comment:
I don't think it's clear that capitalized B or lowercase b refers to `Bitcoin`

Reply:
Also let's use `capitalized` with `z` for consistency across the guide (US vs UK)

Reply:
US English another thing for the list! 😅